api.js

```diff
@@ -513,7 +513,6 @@ API.prototype.recordLogEvent = function recordLogEvent(logEvent = {}) {
     )
     return
   }
-  logEvent.message = applicationLogging.truncate(logEvent.message)
```

Comment:
can you please add a test with a very long message `test/unit/api/api-record-log-events.test.js`